features/kata/operator.feature

```diff
@@ -0,0 +1,12 @@
+Feature: kata operator related tests
+  Background:
+  Given Valid cluster type for kata tests exists
+
+  # @author valiev@redhat.com
+  # @case_id OCP-41813
+  @admin
+  @destructive
+  Scenario: operator successfully installed
+    Given Catalog source "redhat-operators" exists in "openshift-marketplace" namespace
```

Comment:
@valtair1891 in cucumber, we don't cap the first word only.  So it should be `Given catalogsource`

Reply:
done